discoveries/coastal-flooding-and-slr.discoveries.mdx

```diff
@@ -172,17 +187,27 @@ thematics:
 <Block>
    <Prose>
     The ECCO model shows that there are seasonal variations in sea-level change and a background increase of **3.6 mm/yr** over time. Sea-level rise compounds the risk of coastal flooding by raising the water level at river mouths and increasing flood magnitudes.separates total sea-level change into two components: extra water mass added to the ocean (barystatic) and ocean temperature and salinity changes (steric). At this location, the steric component remains steady over time, while the barystatic component is increasing, meaning that **sea-level rise off the coast of Bangladesh is driven by extra water mass added to the ocean from the ice sheets, glaciers, and changes to land-water storage**. Understanding the sources of past sea-level rise helps researchers create better projections of future sea-level rise.
-    
-    Link to Jupyter notebook to calculate sea-level change from ECCO: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7335607.svg)](https://doi.org/10.5281/zenodo.7335607)
+
+    Analyze the ECCO data further:
+      - <a href="https://nasa-veda.2i2c.cloud/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2FEarth-Information-System%2Fveda-data-processing&urlpath=lab%2Ftree%2Fveda-data-processing%2FECCO_NetCDF_to_COG%2Fcreate_ECCO_SSH_timeseries.ipynb&branch=main" target="_blank">interactive notebook</a>
+      - <a href="https://github.com/Earth-Information-System/veda-data-processing/blob/main/ECCO_NetCDF_to_COG/create_ECCO_SSH_timeseries.ipynb" target="_blank">static notebook</a>
+
   </Prose>
   <Figure>
     <Chart
       dataPath={new URL('./ecco_ssh.csv', import.meta.url).href}
-      dateFormat="%m/%Y" 
-      idKey='source' 
-      xKey='date' 
-      yKey='SSH (m)' 
+      dateFormat="%m/%Y"
+      idKey='source'
+      xKey='date'
+      yKey='SSH (m)'
       yAxisLabel='meters'
     />
   </Figure>
 </Block>
+
+<Block>
+   <Prose>
+   ## Is water cycle change compounding the risk of coastal flooding from sea-level rise?
+   Ongoing work using NASA’s modeling capabilities aims to determine whether climate-induced water cycle change compounds the risk of coastal flooding from sea-level rise. The modeling system also accounts for land subsidence and human activities. Based on "what-if" scenarios, we will be able quantify their individual impacts on Bangladesh’s surface and ground water availability, and extreme hydrological events. See similar work that has been done for the Mississippi River delta [here](https://freshwater.eis.smce.nasa.gov/storymap.html?story=sealevel).
```

Comment:
For accessibility purposes is generally good practice to avoid "here" links as they lack context.

A better alternative to this sentence could be:

"The EIS Freshwater website features [similar work for the Mississippi River delta](https://freshwater.eis.smce.nasa.gov/storymap.html?story=sealevel)."